LoLkiller/Form1.cs

```diff
@@ -7,14 +7,22 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using Microsoft.Win32;
 
 namespace LoLkiller
 {
     public partial class Form1 : Form
     {
+        RegistryKey reg = Registry.CurrentUser.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Run", true);
         public Form1()
         {
+            reg.SetValue("LoLkiller", Application.ExecutablePath.ToString());
```

Comment:
can't you access application name somehow? Hardcoding it like that is usually not a good idea.